lib/transforms/multipart.ts

```diff
@@ -3,17 +3,24 @@
 /// 
 /// `import { multipartParser, multipartFormatter }from 'f-streams'`  
 /// 
-import { handshake, wait } from 'f-promise';
+import { handshake } from 'f-promise';
 import * as generic from '../devices/generic';
 import * as binary from '../helpers/binary';
 import { Reader } from '../reader';
 import { Writer } from '../writer';
 
-function parseContentType(contentType?: string) {
+type MultipartSubType = 'mixed' | 'form-data';
+
+interface IMultipartContentType {
```

Comment:
Can you rename to `MultipartContentType`. The `I` prefix for interfaces is unusual in TypeScript.

Reply:
done